src/helpers/QuotaHelpers.ts

```diff
@@ -1,10 +1,9 @@
-import { Jadwal } from 'data/types';
+import { Jadwal, KuotaRt } from 'data/types';
 
 export function hasQuota(jadwal: Jadwal[] = []) {
   for (const jadwalItem of jadwal) {
-    for (const waktuItem of jadwalItem.waktu ?? []) {
-      const { kuota = {} } = waktuItem;
-      if (kuota.jakiKuota || kuota.sisaKuota || kuota.totalKuota) {
+    for (const waktuItem of jadwalItem.waktu) {
+      if (KuotaRt.validate(waktuItem.kuota).success) {
```

Comment:
does this also validates that each kuota > 0?

Previously the logic uses boolean cast, but the actual intention is to handle < 0

Reply:
oh ya, this one is missing :( will update revert it to the latest one. It just check the existence of the fields.